src/zope/server/http/wsgihttpserver.py

```diff
@@ -136,19 +137,24 @@ def executeRequest(self, task):
             for value in result:
                 task.write(value)
         except:
-            import sys, pdb
-            print("%s:" % sys.exc_info()[0])
-            print(sys.exc_info()[1])
-            zope.security.management.restoreInteraction()
-            try:
-                pdb.post_mortem(sys.exc_info()[2])
-                raise
-            finally:
-                zope.security.management.endInteraction()
+            self.post_mortem(sys.exc_info())
         finally:
             if hasattr(result, "close"):
                 result.close()
 
+    @classmethod
+    def post_mortem(cls, exc_info):
+        import pdb
+        print("%s:" % exc_info[0])
+        print(sys.exc_info()[1])
```

Comment:
I think you want `exc_info`, not `sys.exc_info()` here.

Reply:
Personally I would also prefer something like

```python
print("{0.__class__.__name__}: {1}".format(*exc_info))
```

(in terms of output formatting, not necessarily this exact spelling.)

Reply:
Good point about the variable vs method call. That was missed during the refactoring.

In practice, this is called from an exception handler so it doesn't make a difference right now, but it makes sense to fix it.